private-templates-service/client/src/components/AllCards/AllCards.tsx

```diff
@@ -34,6 +36,7 @@ export class AllCards extends Component<Props> {
         getTemplates={this.props.getTemplates}
         pageTitle={ALL_CARDS_TITLE}
         pageID={ALL_CARDS}
+        basePath={"/templates/all"}
```

Comment:
are we localizing url paths?

Reply:
not really but it seems that there's some logic in other places based on this string, if someone changes this string in one place, it could break other things, that's why it might be a good approach to declare and export it from one place

`export const allTemplate = "/templates/all"`